docs/GettingStarted.asciidoc

```diff
@@ -264,6 +264,52 @@ Each click point can have an `id`, and if a needle contains multiple click point
 you must pass it to `testapi::assert_and_click` to select which click point
 to use.
 
+=== Configuration
```

Comment:
I wonder why you put this section into "GettingStarted" and not "Installing". Especially because we now by default do not need files like openqa.ini and such anymore for default operation I think it's better suited to move this section to Installing and keep this documented tailored for users which might not even need to maintain their own openQA installation.

Reply:
I added it to "GettingStarted" because this document is the first and already contains mentionings of `/etc/openqa/openqa.ini`. So I thought it makes most sense to explain the concept of configuration upfront as it is a prerequisite for following sections.

I must also say that I find the configuration in general a very fundamental thing to know about when using software. It seems actually more important than some of the other topics we mention on "GettingStarted".

I can nevertheless move it to "Installing" if you like that better.

Reply:
Open to hear feedback from others. As this is about documentation also "beginners" can provide helpful insight

Reply:
Maybe @realcharmer could provide some useful feedback here?

Reply:
Whilst looking at it as a relative newcomer, I prefer the way @Martchus structures it. I'd like to know what each configuration file does before diving into installation itself. I might also already be using an existing instance without having to install it myself from scratch. Having an initial overview of the config file is good in my opinion, although i do understand what @okurz is trying to point out.